syntax-tests.sh

```diff
@@ -58,6 +58,20 @@ link_package() {
     ln -vs "$(realpath "$INPUT_PACKAGE_ROOT")" "$packages/$INPUT_PACKAGE_NAME"
 }
 
+link_additional_packages() {
+    if [[ $INPUT_ADDITIONAL_PACKAGES != false ]]; then
+        IFS=","
+        for pkg in $INPUT_ADDITIONAL_PACKAGES; do
+            # link additional package into testing dir's Package folder
+            ln -vs "$(realpath "$pkg")" "$packages/$pkg"
```

Comment:
```suggestion
            ln -vs "$(realpath "$pkg")" "$packages/$(basename "$pkg")"
```

To allow for actual paths to third-party package roots.